editor/src/components/canvas/controls/select-mode/select-mode.spec.browser.ts

```diff
@@ -0,0 +1,338 @@
+import { act, fireEvent } from '@testing-library/react'
+import * as TP from '../../../../core/shared/template-path'
+import { wait } from '../../../../utils/test-utils'
+import { makeTestProjectCodeWithSnippet, renderTestEditorWithCode } from '../../ui-jsx.test-utils'
+import { CanvasControlsContainerID } from '../new-canvas-controls'
+
+describe('Select Mode Selection', () => {
+  beforeAll((done) => {
+    // we need to set the Electron window to a larger size so document.elementsUnderPoint works correctly!
+    const currentWindow = require('electron').remote.getCurrentWindow()
+    const size = currentWindow.getSize()
+    if (size.width !== 2200) {
+      currentWindow.once('resize', () => {
+        done()
```

Comment:
You need to add a second `done()` in an `else` clause otherwise it will never get called if the window doesn't need resizing

Reply:
oh thanks!

Reply:
fixed!